Resources/workshops.md

```diff
@@ -0,0 +1,12 @@
+# Workshops
+
+## Web Workshop
+
+* [Github Repo](https://github.com/Shopify/web-workshop)
+
+This is a 5 hour workshop originally written for RnD Summit 2018. The workshop begins by covering the high-level priciples of the web-foundation team, it then moves on to demonstrate their practical application while coding a private Shopify App that communicates with the Shopify Admin GraphQL API. With a mix of lectures, individual exercises, and coding together, the workshop covers the core languages of the Web stack (React, Typescript, GraphQL) as well as our internal libraries (Sewing Kit, Polaris, Quilt, WebGen).
```

Comment:
I would just strip out webgen here, it’s unlikely to be the name we use long term